### PR TITLE
Production cleanup

### PR DESCRIPTION
**Pull request by abdulmhamid**
Let's get these merged into master.  The listed changes cover most of the to-dos.

------
Todo list for adding configurable behaviors, e.g. use Sass, use ember-responsive breakpoints, etc.

- [x] Rename component to `unified-select` (completed on ah/production-cleanup branch) - AH
- [-] Use getOwner to looking config:environment for addon to utilize consuming app config values 
   - no longer necessary
- [ ] Setup example dummy app config for opt-in use of sass
- [-] Setup config for defining breakpoints, e.g. defaults or map to consuming app breakpoint names - BH
   - no longer necessary
- [x] Cleanup references to prior code -AH
- [x] Create substitute for icons and their usage -AH
- [~] Setup demo on github pages showing default styles -AH
   - dummy app in progress, will work on getting page set up when these changes are merged

### CODE REVIEW

File: public/images/chevron-down-icon.svg [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 330 330" style="enable-background:new 0 0 330 330;" xml:space="preserve">
+<path id="XMLID_225_" d="M325.607,79.393c-5.857-5.857-15.355-5.858-21.213,0.001l-139.39,139.393L25.607,79.393
+	c-5.857-5.857-15.355-5.858-21.213,0.001c-5.858,5.858-5.858,15.355,0,21.213l150.004,150c2.813,2.813,6.628,4.393,10.606,4.393
+	s7.794-1.581,10.606-4.394l149.996-150C331.465,94.749,331.465,85.251,325.607,79.393z"/>
+<g>
```
Comment (hergaiety) [inline public/images/chevron-down-icon.svg:8]: Minor thing, but all these `<g>`'s can go away haa.

File: tests/integration/components/unified-select-test.js [hunk 2]
```diff
@@ -4,185 +4,186 @@ import { setupComponentTest } from 'ember-mocha';
...
-        this.render(hbs `{{select-box selectId="sbNativeFlat" options=options selected=selected nativeSelect=true}}`);
+        this.render(hbs `{{unified-select selectId="sbNativeFlat" options=options selected=selected nativeSelect=true}}`);
         expect(find('#sbNativeFlat').value).to.eq('3');
     });
     it('clears query string on click of close icon', function () {
-        this.set('media', {isDesktop: false});
         this.set('label', 'Label');
         this.set('dropdownOpen', true);
-        this.set('showMobileSearch', true);
+        this.set('showModalSearch', true);
```
Comment (hergaiety) [inline tests/integration/components/unified-select-test.js:79]: I like this rename a lot.

**Review by hergaiety — APPROVED** [review]
Looks good, I skimmed a lot of it since it's mostly renaming of files.

**Comment by hergaiety** [comment]
Does this resolve #2? If so we can tie this PR to the issue directly I believe by commenting as such in the PR's description.

**Comment by abdulmhamid** [comment]
Not fully, so I don't want to close it quite yet.

**Pull request merged**